src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -200,7 +200,6 @@ http://purl.obolibrary.org/obo/MONDO_0007672	http://identifiers.org/hgnc/14373	O
 http://purl.obolibrary.org/obo/MONDO_0007681	http://identifiers.org/hgnc/17098	OMIM:138800	https://omim.org/entry/606241
 http://purl.obolibrary.org/obo/MONDO_0007686	http://identifiers.org/hgnc/31928	OMIM:139090	https://omim.org/entry/614169
 http://purl.obolibrary.org/obo/MONDO_0007688	http://identifiers.org/hgnc/6770	OMIM:139210	https://omim.org/entry/600993
-http://purl.obolibrary.org/obo/MONDO_0007690	http://identifiers.org/hgnc/2594	OMIM:139300	https://omim.org/entry/107910
```

Comment:
I confirmed this OMIM https://omim.org/entry/139300 no longer matches the rules for disease defining.

Reply:
Hmm, I'm debugging the OMIM pipeline and it's currently not even processing OMIM:139300 or OMIM:107910 to add associations for them. It's possible that the data has changed since your build was run, but that would be surprising since the OMIM release ran just 4 days ago. Won't be able to look into this until next week.

Reply:
The removal of the disease defining association is correct.

Reply:
When did you run this? I just downloaded the file from [the release](https://github.com/monarch-initiative/omim/releases/tag/2025-02-09) and this entry is not there.

If you didn't run this before Sunday, then maybe there is some problem with `mondo-ingest` fetching old data.